src/unix/linux_like/linux/musl/b32/hexagon.rs

```diff
@@ -286,7 +286,7 @@ pub const SYS_clock_settime: c_int = 112;
 pub const SYS_clone: c_int = 220;
 pub const SYS_close: c_int = 57;
 pub const SYS_connect: c_int = 203;
-pub const SYS_copy_file_range: c_int = -1; // FIXME
+pub const SYS_copy_file_range: c_int = -1; // FIXME(musl)
```

Comment:
I don't know if there is something musl-specific here, probably `FIXME(hexagon)` in this file